packages/ui-tree-browser/src/TreeBrowser/TreeNode/__tests__/TreeNode.test.js

```diff
@@ -32,7 +32,7 @@ import {
 } from '@instructure/ui-test-utils'
 
 import { TreeNode } from '../index'
-import styles from '../../TreeButton/styles.css'
```

Comment:
Do you know why it messes up the theming?

Reply:
yeah, I forgot to change here, it broke the test, but I've fixed it :) (the classes are generated with hash, and it was looking for the buttons css hash instead of the node css hash)